rotkehlchen/chain/bitcoin/utils.py

```diff
@@ -21,18 +30,90 @@ class OpCodes:
     op_checksig = b'\xac'
 
 
+class BitcoinSegwitEncoding(Enum):
+    """Enumeration type to list the various Bitcoin Segwit supported encodings."""
+    BECH32 = 1  # Native Segwit(P2WPKH)
+    BECH32M = 2  # Taproot(P2TR)
+
+
 def is_valid_btc_address(value: str) -> bool:
-    return is_valid_base58_address(value) or is_valid_bech32_address(value)
+    return (
+        is_valid_base58_address(value) or
+        is_valid_bech32_address(value) or
+        is_valid_bech32_bech32m_address(value)
```

Comment:
Aha, I think I got it. Thank you for the explanation!

(maybe also will make sense to add a bit of this text in the code since I personally was a bit confused when found really little difference between these methods and methods from the `bech32` sourcecode)